staff/sergiomons/css-layout-simple-adaptive/styles/desktop.css

```diff
@@ -0,0 +1,34 @@
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+ }
+ html, body {
+    min-height: 100% !important;
+    height: 100%;
+    background-color: black;
+    color:white;
+    text-align:center;
+}
+header, footer{
+  width:100%;
```

Comment:
TIP is ```width: 100%```rule required?

Reply:
Ok, I think it's not necessary. I,m going to delete it.